reactnative/RNPlayground/app/screens/AboutScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackScreenProps } from "../navigators"
+import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import {
+  Text, Screen
+} from "../components"
+import { isRTL } from "../i18n"
+import { colors, spacing } from "../theme"
+import { useSafeAreaInsetsStyle } from "../utils/useSafeAreaInsetsStyle"
+
+export const AboutScreen: FC<StackScreenProps<AppStackScreenProps, "About">> = observer(function AboutScreen() {
+    const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
+
+    return (
+      <View style={$container}>
+        <View style={$topContainer}>
+          <Text
+            testID="welcome-heading"
+            style={$welcomeHeading}
+            tx="aboutScreen.readyForLaunch"
+            preset="heading"
+          />
+          <Text tx="aboutScreen.postscript" preset="subheading" />
```

Comment:
Is this alright? shouldn't be `<Text tx={aboutScreen.postscript} preset="subheading" />` ?